src/keri/vc/proving.py

```diff
@@ -25,8 +22,9 @@ def credential(schema,
                issuer,
                data,
                recipient=None,
-               private=False,
-               salt=None,
+               private:bool=False,
+               private_credential_nonce:Optional[str]=None,
+               private_subject_nonce:Optional[str]=None,
```

Comment:
If you are going to introduce type hints to parameters in a method (which is a good thing), please update the entire signature of the method, not just the few you are touching.

Reply:
my bad!